src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -118,7 +129,7 @@
         [Option(
             "max-file-size-in-kb",
             HelpText = "The maximum file size (in kilobytes) that will be analyzed.",
-            Default = 1024)]
-        public int MaxFileSizeInKilobytes { get; set; } = 1024;
+            Default = -1)]
```

Comment:
>-1 [](http://example.com/codeflow?start=22&length=2)

the benefit of have 1024 in the CommonOption here is it will be shown to user. with changed to -1, it is actually 1024 as we coded inside c# but just is not as clear to user.